tasks/create_http_task_with_token/create_http_task_with_token.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START cloud_tasks_create_http_task_with_token]
+
+// Command createHTTPTaskWithToken constructs a task with a authorization token
```

Comment:
This is just a package, not a command/binary. So, this would be `Package token constructs...`. See https://github.com/golang/go/wiki/CodeReviewComments#package-comments.